.github/workflows/configuration.json

```diff
@@ -1,16 +1,59 @@
 {
   "categories": [
+    {
+      "title": "### Autotuner",
+      "labels": ["autotuner"],
+      "exhaustive": true
+    },
+    {
+      "title": "### QualX",
+      "labels": ["qualx"],
+      "exhaustive": true
+    },
+    {
+      "title": "### API Change",
+      "labels": ["api_change"],
+      "exhaustive": true
+    },
+    {
+      "title": "### Feature Request",
+      "labels": ["feature request"],
+      "exhaustive": true
+    },
+    {
+      "title": "### Performance",
+      "labels": ["performance"],
+      "exhaustive": true
+    },
+    {
+      "title": "### Build and CI/CD",
+      "labels": ["dependencies", "cicd", "build"],
+      "exhaustive": true
+    },
+    {
+      "title": "### Bug Fixes",
```

Comment:
Previously there would be very few overlaps i.e. when a PR is labelled as both core tools and user tools.

However, since we have `bug` and `feature request` along with specific areas, that might lead to many duplicate rows in Changelog. Would that be an issue?